ecosystem/indexer-grpc/indexer-transaction-generator/src/tests/README.md

```diff
@@ -0,0 +1,3 @@
+# Testing Files for the framework.
+
+__Do not put actual test case here__
```

Comment:
Maybe mention what they should do instead.

Reply:
done.